benchmark/gcbench/conmsg.d

```diff
@@ -16,7 +16,7 @@ JSONValue buildVal(in dchar[] word)
     foreach (i; 0 .. pos.length)
         pos[i] = i;
     res["array"] = pos;
-    return JSONValue(res);
+    return cast(immutable) JSONValue(res);
```

Comment:
Can the function be made `pure` instead of having to cast to immutable?

Reply:
I didn't know that would do that. Great! So done.

Reply:
If it's a `pure` function that compiler can infer that a value is unique and therefore implicitly convertible to `immutable`.